src/plugins/console/server/lib/spec_definitions/json/generated/cat.fielddata.json

```diff
@@ -5,7 +5,7 @@
       "bytes": [
         "b",
         "k",
-        "osd",
+        "kb",
```

Comment:
This 💯 looked like a regression to me, and then I took a second look at what this array is... 😂

Reply:
Yeah it was ha.